StorageUtility/Windows/Device.cpp

```diff
@@ -44,7 +44,7 @@ DeviceHandle cDevice::Handle()
     eErrorCode errorCode;
     DeviceHandle deviceHandle;
 
-    errorCode = GetStorageDeviceHandle(m_DevDetailData->DevicePath, deviceHandle);
+    errorCode = GetStorageDeviceHandle(m_DevDetailData->DevicePath, deviceHandle.Handle);
```

Comment:
Function GetStorageDeviceHandle should take DeviceHandle input instead of HANDLE, you forget to assign initial value for bustype also?